src/containers/AlbumsView.jsx

```diff
@@ -15,13 +15,20 @@ export class AlbumsView extends Component {
   constructor (props) {
     super(props)
     this.state = {isFetching: true, error: false}
+
+    this.handleErrorFromChildren = this.handleErrorFromChildren.bind(this)
+
     props.fetchAlbums()
       .then(() => { this.setState({isFetching: false, error: false}) })
       .catch(albumsError => {
         this.setState({isFetching: false, error: albumsError})
       })
   }
 
+  handleErrorFromChildren (error) {
```

Comment:
I am open about this point, I am not a big of the current naming as well, I just suggested here.